docs/how-to/evaluate/custom_dataloader.md

```diff
@@ -0,0 +1,9 @@
+# How to create custom DataLoader for Ragbits evaluation
+
+Ragbits provides a base interface for data loading, `ragbits.evaluate.loaders.base.DataLoader`, designed specifically for evaluation purposes. A ready-to-use implementation, `ragbits.evaluate.loaders.hf.HFLoader`, is available for handling datasets in huggingface format.
+
+To create a custom DataLoader for your specific needs, you need to implement the `load` method in a class that inherits from the `DataLoader` interface.
+
+Please find the [working example](optimize.md#define-the-data-loader) here.
```

Comment:
I don't think linking to the optimizer is enough here, we should have a standalone example going in-detail about data loaders.

Reply:
Resolved in https://github.com/deepsense-ai/ragbits/pull/217/